jmix-flowui/flowui/src/main/java/io/jmix/flowui/view/builder/DetailWindowBuilderProcessor.java

```diff
@@ -111,7 +108,7 @@ protected <E, V extends View<?>> void setupListDataComponent(DetailWindowBuilder
                 E entityFromDetail = getSavedEntity(detailView, parentDataContext);
                 E reloadedEntity = transformForCollectionContainer(entityFromDetail, container);
                 E savedEntity = transform(reloadedEntity, builder);
-                E mergedEntity = merge(savedEntity, builder.getOrigin(), parentDataContext);
+                E mergedEntity = merge(savedEntity, builder.getOrigin(), parentDataContext, container);
```

Comment:
The same change is needed for the `DetailViewBuilderProcessor` from the Tabbed Application Mode add-one.